src/pages/ProductList/Component/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './Dropdown.scss';
+import DropdownList from './DropdownList';
+import DropdonwItem from './DropdownItem';
+
+const Dropdown = props => {
+  const [dropdownVisibility, setDropdownVisibility] = React.useState(false);
```

Comment:
- 개발자가 커스텀하게 만드는 custom hook과 구분짓기 위해 이런 방식으로 useState를 사용하기도 하지만,
일반적으로는 import할 때 `import React, { useState } from 'react';` 와 같이 import해서 사용합니다!
- boolean naming convention에 맞게 앞에 is를 붙여서 네이밍해주세요!
```suggestion
  const [isDropdown, setIsDropdown] = useState(false);
```

Reply:
수정 하였습니다.